jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/SortingOverviewUnitTests.java

```diff
@@ -0,0 +1,32 @@
+package io.github.epam.angular.tests.unit;
+
+import io.github.com.pages.AngularPage;
+import io.github.com.pages.sections.SortingOverviewSection;
+import io.github.epam.TestsInit;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class SortingOverviewUnitTests extends TestsInit {
+
+    @BeforeMethod(alwaysRun = true)
+    public void before() {
+        AngularPage angularPage = new AngularPage();
```

Comment:
Why do we need to create this object here?

Reply:
fixed